CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+Contributing
```

Comment:
Shouldn't this include the Apache license header?

Reply:
Why?

I don't think it's reasonable to include a license header in documentation files. If we want every page to include a reference to the license, it should be injected during the build of the page.

Reply:
I guess it seems inconsistent that we do this for code but not docs. I do agree that the license makes it clear that "unless otherwise indicated, all the content of this repository is distributed under the Apache License 2.0". So I'm fine with leaving it out.